ExternalModule.php

```diff
@@ -42,14 +42,17 @@ function redcap_every_page_top($project_id) {
         }
     }
 
+    // Because REDCap does not recognize custom action tags this block appends 
+    // the REDCap action tag @DEFAULT in the case that any custom actions tags (@DEFAULT-*)
+    // have been applied to a field.
     function redcap_survey_page_top($project_id) {
         if ( !$this->getProjectSetting('use_in_survey') ) return;
         global $elements;
         // set the action_tag_class as it would be in the DataEntry context
         foreach( $elements as &$element) {
             $i = array_search($element['name'], $this->survey_APF_fields);
             if ( $i !== false ) {
-                $element['action_tag_class'] = '@DEFAULT';
+                $element['action_tag_class'] .= " @DEFAULT"; // append to preserve existing tags. This can result in duplicate @DEFAULT action tags
```

Comment:
```suggestion
                // append to preserve existing tags. This can result in duplicate @DEFAULT action tags but there appear to be no affects from this
                $element['action_tag_class'] .= " @DEFAULT";
```